### DISCUSSION TITLE
Where is the custom prompt stored?

### DISCUSSION THREAD

**Discussion by shenweizju** [logancyang/obsidian-copilot#516] [Q&A]
When I switch to another macOS, the custom prompt is not synced, all the other files and settings are synced, so I'm confused about the location where the custom prompt stores.

**Comment by logancyang** [comment]
It's in Obsidian's local indexedDB. Some folks don't want the custom prompts to be synced across personal vs. work devices. But a better option might be to allow sync if needed. There's a FR for it.

**Reply by TangentFoxy** [reply]
Aren't they now stored in `vault root/copilot/copilot-custom-prompts`?
